core/migrations/14_deploy_expiring_multi_party_creator.js

```diff
@@ -17,6 +18,10 @@ module.exports = async function(deployer, network, accounts) {
   const finder = await Finder.deployed();
   const tokenFactory = await TokenFactory.deployed();
 
+  // Deploy EMPLib and link to EMPCreator.
+  await deploy(deployer, network, ExpiringMultiPartyLib);
+  await deployer.link(ExpiringMultiPartyLib, ExpiringMultiPartyCreator);
```

Comment:
Is there another example in the repo of linking libraries with contracts before deploying? I couldn't find one so I did it explicitly here.

Reply:
If you look at the `TokenizedDerivativeUtils` library from before we removed it from the repo, we did the same thing there.